src/createRouter.ts

```diff
@@ -121,6 +143,23 @@ const createRouter: RouterFactory = compose<RouterMixin, RouterOptions>({
 		}
 	},
 
+	observeHistory(history: History, context: Context, dispatchInitial: boolean = false): PausableHandle {
+		const router: Router = this;
+		if (historyMap.has(router)) {
+			throw new Error('observeHistory can only be called once');
+		}
+		const listener = pausable(history, 'change', (event: HistoryChangeEvent) => {
+			router.dispatch(context, event.value);
```

Comment:
We can worry about this when we actually have experience using navigation deferral though.